iacrcc/tests/test15/main.tex

```diff
@@ -0,0 +1,46 @@
+\documentclass[version=submission,notanonymous]{iacrcc}
+
+\license{CC-by}
+
+\title[running  = {The iacrcc class},
+       onclick  = {https://github.com/IACR/latex},
+       subtitle = {A Template}
+      ]{An example that is not anonymous}
+
+\addauthor[orcid    = {0000-0003-1010-8157},
+           inst     = {1},
+           onclick  = {https://www.joppebos.com},
+	   footnote = {This is an example fö{\"o}tnote.},
+           email    = {joppe.bos@nxp.com}
+	  ]{Joppe W. \surname{B{\"o}s}\newline}
```

Comment:
What happens if someone uses \\\\ instead of \\newline? Also what about the version \\\\*[10pt] which is equivalent to \\\\ but inserts a vertical space of 10pt?

Reply:
By looking at [this page](https://tex.stackexchange.com/questions/175482/how-to-run-ifsubstr-on-a-macro-parameter-that-may-contain-a-newline?rq=1) it seems that one solution is to use
```
\noexpandarg\IfSubStr{#2}{\\}{%                                                                    
    \ClassError{iacrcc}{%                                                                            
      ^^J%                                                                                           
      Do not put a newline in the \string\addauthor\space macro%                                     
      ^^J%                                                                                           
    }{}%                                                                                             
  }{}
```
but we still need to check for \\newline.

Reply:
I tested inserting a newline with \\ but then the author will get some other LaTeX error. 

 Use of \\addauthor doesn't match its definition.
\new@ifnextchar ...served@d = #1\def \reserved@a {
                                                  #2}\def \reserved@b {#3}\f...

The test above does not seem to catch this.

Reply:
I meant that we need to check for both \\\\ and \\newline.

Reply:
I reworked this and this is not working: will create a new pull requests with additional tests and the fix.